axelrod/match.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+
+def sparkline(actions):
```

Comment:
Could we make sparklines take any character? Using `█` by default?

This would also allow us to keep the current doctest which is skipped and include another one where we use `X` (or whatever).